core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -421,11 +419,19 @@ private void cancelNameResolverBackoff() {
     }
   }
 
-  // Must be run from syncContext
-  private void refreshNameResolutionNow() {
+  /**
+   * Force name resolution refresh to happen immediately and reset refresh back-off. Must be run
+   * from syncContext.
+   */
+  private void forceNameResolutionRefresh() {
```

Comment:
"force" and "now" are both not great names, since `refreshNameResolution` always refreshes. Maybe `refreshAndResetNameResolution`? Although that's not perfect, at least people may read the javadoc instead of assuming they know what it means. Dunno.